robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro

```diff
@@ -10,7 +10,7 @@
   <xacro:include filename="$(find robotino_description)/urdf/arm.urdf.xacro" />
   <xacro:include filename="$(find softhand_description)/model/softhand_withCableCanal.urdf.xacro" />
 
-  <xacro:base name="base" has_8dof="false"/>
+  <xacro:base name="base" has_8dof="true"/>
```

Comment:
@Senka2112 please verify that this is correct for your robot , check if on your robot version is also has_8dof set tot true